zulipterminal/ui_tools/views.py

```diff
@@ -331,18 +331,28 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
 
         elif is_command_key('REPLY_MESSAGE', key):
             self.body.keypress(size, 'enter')
+            if self.footer.focus is None:
+                return key
             self.set_focus('footer')
             self.footer.focus_position = 1
             return key
 
         elif is_command_key('STREAM_MESSAGE', key):
             self.body.keypress(size, 'c')
+            stream_id = self.model.stream_id
+            stream_dict = self.model.stream_dict
```

Comment:
These two lines are only used in the conditional (`if` statement), so can be moved into there.